clean.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+REM Copyright (c) .NET Foundation and contributors. All rights reserved.
+REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if "%~1"=="" (
+  @call run.cmd clean -?
+) else (

```

Comment:
Added the exit call after the `clean -?` call; however, I think the `else` statement adds value since it provides clarity about when the commands are supposed to run